pyomo/solvers/tests/checks/test_no_solution_behavior.py

```diff
@@ -35,6 +35,7 @@
 # A function that returns a function that gets
 # added to a test class.
 #
+@unittest.nottest
```

Comment:
Why is this decorator necessary?

Reply:
Because recent versions of nose find these functions whose name starts with test.  I'm not sure why.